src/routes/google-auth/index.js

```diff
@@ -0,0 +1,46 @@
+import passport from 'passport'
+import express from 'express'
+
+export default function ({db}) {
+  const router = express.Router()
+
+  router.get('/', passport.authenticate('google-login', {
+    scope: ['openid', 'profile', 'email']
```

Comment:
Shouldn't need the email scope once you switch to using ids.

Reply:
Resolved with https://github.com/Wingysam/Christmas-Community/pull/138/commits/607b7887945799def64dee7415b2b739b82f4151